src/components/oidc/OidcConfig.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { OidcConfiguration } from '@axa-fr/react-oidc/dist/vanilla';
+
+export const onEvent = (configurationName: string, eventName: string, data: object) => {
+    console.log(`oidc:${configurationName}:${eventName}`, data);
+};
+
+export const OidcConfig: OidcConfiguration = {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    authority: process.env.REACT_APP_ZITADEL_AUTHORITY_NAME!,
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    client_id: process.env.REACT_APP_ZITADEL_CLIENT_ID!,
+    redirect_uri: window.location.origin + '/authentication/callback',
```

Comment:
/authentication/callback and /authentication/silent-callback also must be moved to .env file